scalarm_client_csharp/Client.cs

```diff
@@ -26,6 +27,42 @@ public Client(string baseUrl)
             // this.AddHandler("application/json", new JsonConvertDeserializer());
 		}
 
+		//absolutePath must be to file with extension .tar.gz
+		public string GetSimulationScenarioFiles(string scenarioId, string absolutePath)
+		{
+			//var path = Environment.CurrentDirectory;
+			var request = new RestRequest("/simulation_scenarios/{id}/code_base", Method.GET);
+			request.AddUrlSegment("id", scenarioId);
+			IRestResponse restResponse = this.Execute (request);
+			if (restResponse.ErrorException != null) {
+				const string message = "Error retrieving response.  Check inner details for more info.";
+				var ResponseException = new ApplicationException (message, restResponse.ErrorException);
+				throw ResponseException;
+			} else {
+				var code = restResponse.RawBytes;
+				code.SaveAs(absolutePath);
+				return "OK";
+			}
+		}
+		//absolutePath must be to file with extension .tar.gz
+		public string GetExperimentBinaryFiles(string scenarioId, string absolutePath)
```

Comment:
@Andriu01 - nie zwróciłem na to uwagi wcześniej - ale tą metodę można spokojnie przenieść do Experiment. Albo coś źle powiedziałem wcześniej (?)